docs/source/classification/jaccard_index.rst

```diff
@@ -10,11 +10,57 @@ Jaccard Index
 Module Interface
 ________________
 
+CohenKappa
+^^^^^^^^^^
+
 .. autoclass:: torchmetrics.JaccardIndex
     :noindex:
 
+BinaryJaccardIndex
+^^^^^^^^^^^^^^^^^^
+
+.. autoclass:: torchmetrics.BinaryJaccardIndex
+    :noindex:
+    :exclude-members: update, compute
+
+MulticlassJaccardIndex
+^^^^^^^^^^^^^^^^^^^^^^
+
+.. autoclass:: torchmetrics.MulticlassJaccardIndex
+    :noindex:
+    :exclude-members: update, compute
+
+MultilabelJaccardIndex
+^^^^^^^^^^^^^^^^^^^^^^
+
+.. autoclass:: torchmetrics.MultilabelJaccardIndex
+    :noindex:
+    :exclude-members: update, compute
+
+
 Functional Interface
 ____________________
 
+jaccard_index
+^^^^^^^^^^^^^
+
 .. autofunction:: torchmetrics.functional.jaccard_index
     :noindex:
+
+binary_jaccard_index
```

Comment:
for a moment I was thinking if the binary/multi-class shall be rather as posix, but in the end, this makes also sense and it is smoother to read

Reply:
Agree. I think there are pros/cons with both ways, so I basically just settled for one